charter.html

```diff
@@ -272,6 +272,8 @@ <h2>Success Criteria</h2>
           <p>There should be testing plans for each specification, starting from the earliest drafts.</p>
 
     <p>Normative specifications which have user-facing features should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and recommendations for maximizing accessibility in implementations.</p>
+          
+          <p>Non-browser-based functionality in any feature should be clearly established in such a way that any entity who can fill that role in the ecosystem is permitted to do so.</p>
```

Comment:
It does seem like it may open that as a possibility. However, some have taken the position that specifications should require UAs to not be able to be discriminatory in the selection of entities that fill roles like IPA helper servers as long as they meet the technical requirements of the spec. 

Personally, I'm not sure that makes sense or follows how specs have worked in the past generally, but it might be useful to speak to why the capacity to exercise judgement of entities is important.

Reply:
I'll confess that I didn't understand this paragraph, but the added context helps a great deal.

I disagree with this as stated.  The functioning of any system that operates on personal information should have some path whereby the affected persons can exercise a measure of control.  There are various ways to accomplish this, but saying that if the data leaves the user agent, then the user agent cannot have any say in how that data is handled will mean that user agents will be forced to refuse to let any such information leave the user agent.  Otherwise, they will be unable to fulfill their obligation to protect user data.

This requirement would likely foreclose on any system for centralized aggregation, which generally depend on having some system that is trusted to some limited extent to operate on information provided by user agents.  That information would not leave the user agent if it the user agent had no way to control what happens to it.

As to the underlying requirement, I don't understand why this is a concern with the charter.  Why is it that this constraint cannot be debated in the proposed working group?